src/main/java/org/moltimate/moltimatebackend/request/ActiveSiteAlignmentRequest.java

```diff
@@ -29,11 +30,15 @@
     private List<MultipartFile> customMotifs = new ArrayList<>();
     private String ecNumber; // TODO: Make this into a filter
 
-    public List<Structure> getPdbIdsAsStructures() {
-        return ProteinUtils.queryPdb(pdbIds);
+    public List<Structure> callPdbForStructures() {
+        List<Structure> structures = ProteinUtils.queryPdb(pdbIds);
+        if (structures.size() == 0) {
+            throw new InvalidPdbIdException(pdbIds);
+        }
+        return structures;
     }
 
-    public List<Motif> getCustomMotifs() {
+    public List<Motif> convertCustomMotifs() {
```

Comment:
Saying `convert` instead of `get` might imply the object is being mutated. Is there a reason for renaming this?